service/project.go

```diff
@@ -461,7 +461,7 @@ func (uis *UIServer) modifyProject(w http.ResponseWriter, r *http.Request) {
 				uis.LoggedError(w, r, http.StatusInternalServerError, errors.Wrap(err, "can't check if GitHub aliases are set"))
 				return
 			}
-			if !aliasesDefined {
+			if !aliasesDefined && !responseRef.VersionControlEnabled {
```

Comment:
Aren't we still running the risk of there being no aliases defined? While they can defined in YAML, how do we know that it is?

Reply:
I don't think it would be necessary to add any more checks because a PR patch will already fail validation if there is no PR alias defined: https://github.com/evergreen-ci/evergreen/blob/main/units/patch_intent.go#L910

The `FindAliasInProjectRepoOrPatchedConfig ` checks both the project ref and the project YAML for alias definitions.

Reply:
Do we want to allow enabling PR testing though without doing both checks?

Reply:
Yes, because since PRs are their own patches they will need to be validated at the time they are submitted.  If project page PR aliases are empty then Evergreen will next check if PR aliases are defined in the YAML for the patch. If not, the validation error I linked above will hit and the user can just fix it easily by adding PR aliases to their config.

It wouldn't be feasible to check anything else here since we don't know what PRs look like until they are created in the future.  The patch validation will need to happen separately on an individual basis.